### PR TITLE
[CP-2749] [E2E][HomePage] MC Update available - Force Update 

### PR DESCRIPTION
**Pull request by patrycja-paczkowska**
JIRA Reference: [CP-2749](https://appnroll.atlassian.net/browse/CP-2749)

[CP-2749]: https://appnroll.atlassian.net/browse/CP-2749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/mudita-center-e2e/src/page-objects/modal-app-update.page.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import Page from "./page"
+
+class ModalAppUpdate extends Page {
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/page-objects/modal-app-update.page.ts:8]: All modals are extending Page, We should probably change this in a future

File: apps/mudita-center-e2e/src/page-objects/modal-app-update.page.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    return $("h4*=Update Mudita Center to")
+  }
+
+  public get paragraphCurrentVersion() {
+    return $("p*=Update it to use")
+  }
+
+  public get paragraphPrivacyPolicy() {
+    return $(
+      "p*=Please accept the Privacy Policy to start updating Mudita Center."
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/page-objects/modal-app-update.page.ts:23]: Can you make it single line? 
```
      return $("p*=Please accept the Privacy Policy to start updating Mudita Center.")
```

File: apps/mudita-center-e2e/src/specs/overview/e2e-mock-mc-force-update-available.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    const paragraphAvailableVersion =
+      await modalAppUpdatePage.paragraphAvailableVersion
+    await expect(paragraphAvailableVersion).toBeDisplayed()
+    await expect(paragraphAvailableVersion).toHaveTextContaining(
+      "Update Mudita Center to"
+    )
+    const textParagraphAvailableVersion =
+      await paragraphAvailableVersion.getText()
+    const availableAppVersion = textParagraphAvailableVersion.split("to ").pop()
+    await console.log("AVAILABLE VERSION:" + availableAppVersion)
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/overview/e2e-mock-mc-force-update-available.e2e.ts:48]: Please remove console logs

File: apps/mudita-center-e2e/src/specs/overview/e2e-mock-mc-force-update-available.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    // Current version
+    const paragraphCurrentVersion =
+      await modalAppUpdatePage.paragraphCurrentVersion
+    await expect(paragraphCurrentVersion).toBeDisplayed()
+    await expect(paragraphCurrentVersion).toHaveTextContaining(
+      "Update it to use the full version of the Mudita Center. Your current version:"
+    )
+    const textParagraphCurrentVersion = await paragraphCurrentVersion.getText()
+    const currentAppVersion = textParagraphCurrentVersion.split(": ").pop()
+    await console.log("CURRENT VERSION:" + currentAppVersion)
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/overview/e2e-mock-mc-force-update-available.e2e.ts:59]: Please remove console logs

File: apps/mudita-center-e2e/wdio.conf.ts [hunk 5]
```diff
@@ -84,6 +85,7 @@ export const config: Options.Testrunner = {
       toRelativePath(TestFilesPaths.privacyPolicyTest),
       toRelativePath(TestFilesPaths.licenseTest),
       toRelativePath(TestFilesPaths.helpWindowCheckOfflineTest),
+      toRelativePath(TestFilesPaths.mcHomePageForceUpdateTest),
```
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/wdio.conf.ts:88]: This is not a standalone test anymore. We decided everything that uses mocks should be in  `mock: []`

* Kompakt mocks
* Offline mocking
* Force Update / Soft Update MC mocking

**Review by tomaszmaleckimudita — CHANGES_REQUESTED** [review]

**Comment by tomaszmaleckimudita** [comment]
Can't test on Windows, no `.exe` build available. Requested unblocking `Build & deploy to feature branch / build (Windows)`

File: apps/mudita-center-e2e/src/specs/overview/e2e-mock-mc-force-update-available.e2e.ts (hunk 3 above)
Comment (tomaszmaleckimudita) [inline apps/mudita-center-e2e/src/specs/overview/e2e-mock-mc-force-update-available.e2e.ts:48]: expect(availableVersion) to be same as E2EMockClient.setMockUpdateState.version ?

**Review by tomaszmaleckimudita — CHANGES_REQUESTED** [review]

File: apps/mudita-center-e2e/src/page-objects/modal-app-update.page.ts (hunk 2 above)
Comment (patrycja-paczkowska) [reply apps/mudita-center-e2e/src/page-objects/modal-app-update.page.ts:23]: Prettier

**Review by tomaszmaleckimudita — APPROVED** [review]
Works good on all systems

**Pull request merged**